examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -507,6 +507,12 @@ void SetupPretendDevices()
     AddEndpoint("Door 2");
     AddCluster("Door");
     AddAttribute("State", "Closed");
+
+    AddDevice("Occupancy Sensor");
+    AddEndpoint("External");
+    AddCluster("Occupancy Sensor");
+    AddAttribute("Occupancy", "1");
+    chip::app::Clusters::OccupancySensing::Attributes::Occupancy::Set(1, 1);
```

Comment:
```suggestion
    app::Clusters::OccupancySensing::Attributes::Occupancy::Set(1, 1);
```
File has `using namespace ::chip;`.